server/src/routers/converterRouter.js

```diff
@@ -0,0 +1,56 @@
+const express = require('express');
+const multer = require('multer');
+const upload = multer();
+const router = new express.Router();
+
+const { mapping } = require('../constants/config');
+const { parseInputFile, sendResponse } = require('../util');
+
+router.use(express.json());
+
+var Converter = require('api-spec-converter');
+
+//Endpoint for conversion
+router.post('/api/specification/:format/:convertTo', upload.single('file'), (req, res) => {
+    const {format, convertTo} = req.params,
+        {targetSyntax = 'json', toFile = false} = req.query;
+
+    // Check if the conversion is possible
+    if(mapping[format] && mapping[format].includes(convertTo)) {
+      const file = req.file;
+
+      if(!file) {
+        res.status(400).send({'message': 'File not available.'});
```

Comment:
We need to `return` this line.
Right now, even if `file` is null, the function continues to execute - this will lead to errors.